guides/updates/administration/vue3.md

```diff
@@ -0,0 +1,129 @@
+Yes, Vue 3 introduced breaking changes. It's crucial to review the migration guide provided by Vue.js and this document for detailed information.
+
+### What steps should I follow to update my Shopware plugin to Vue 3?
+Typically, the process involves updating your project dependencies and modifying your code to adhere to Vue 3's API changes. Consult the Vue 3 documentation and this document's step-by-step instructions.
+
+### Can one plugin version be compatible with Shopware 6.5 and 6.6?
+Your plugin requires a new version in the Store. For instance, version `1.x` is for Shopware `6.5.x`, while version `2.0` is compatible with Shopware `6.6` and newer.
+
+### How can I check if my Shopware extension is compatible with Vue 3?
+You can verify compatibility by reviewing the extension's functionality and updating test suites according to this document.
+
+### Do I need to rewrite my extension to update to Vue 3?
+While some changes are required, a complete rewrite is not necessary. The amount of effort is dictated by your use of Vue's internal API.
+
+### Are tools or libraries available to facilitate the migration to Vue 3?
+Yes, there are tools and migration helpers that can automate certain aspects of the update process. You could start by enabling the Vue 3 rule set of `eslint`.
+
+### Where can I find support and community discussions about updating Shopware plugin to Vue 3?
+You can participate in discussions and seek help on the Shopware community Slack. There is a dedicated channel for this topic called `#vue3-update`.
+
+## External resources
+
+Here is a handpicked selection of external resources. This list provides a handy reference, granting you access to all the essential materials you might need.
+
+- (Vue 3 migration guide)[https://v3-migration.vuejs.org]
+- (Vue 3 breaking changes)[https://v3-migration.vuejs.org/breaking-changes/]
+- (Vue router migration guide)[https://router.vuejs.org/guide/migration/]
+- (Vue test utils migration guide)[https://test-utils.vuejs.org/migration/]
+
+## Step-by-step guide
+::: warning
+Depending on when you read this document, you still need to enable the `VUE3` feature flag.
+To be safe, enable the `VUE3` feature flag and restart your watcher.
+:::
+
+To follow along, you should have the following:
+
+- the latest Shopware `trunk` or an official release candidate
+- installed and activated your plugin
+- a running administration watcher (`composer run watch:admin`)
+
+### Update your plugin npm dependencies
+
+Make sure to align your `package.json` dependencies with the administration.
+
+### Verify the Admin is using Vue 3
+
+If you evaluate the following expression in the browser dev tools, the value `true` must be returned.
+
+```javascript
+Shopware.Application.view.vue3
+```
+
+### Check your templates
+
+For your templates to work correctly, follow the following in no specific order:
+
+- Replace all `sw-field` usages with the corresponding components.
```

Comment:
Link to the mapping file would be useful, `src/Administration/Resources/app/administration/src/app/component/form/sw-field/index.js` I guess